archetypes/helidon/src/main/archetype/common/docker.xml

```diff
@@ -62,11 +62,11 @@
                     </includes>
                 </templates>
                 <model>
-                    <list key="poms">
-                        <value>pom.xml</value>
+                    <list key="poms" if="!${multi-module}">
+                        <value>pom.xml .</value>
```

Comment:
It might more readable if we put the whole line in the model value. I.e. `ADD src src`.